frontend/src/app/shared/components/work-package-graphs/embedded/wp-embedded-graph.component.ts

```diff
@@ -234,4 +234,31 @@ export class WorkPackageEmbeddedGraphComponent implements OnChanges {
   private isRadarChart() {
     return this.chartType === 'radar' || this.chartType === 'polarArea';
   }
+
+  public get chartSummary():string {
+    const typeLabel = this.chartTypeLabel;
+
+    return `${typeLabel} chart showing work packages which are ${this.chartDescription}.`;
+  }
+
+  public get chartTypeLabel():string {
+    switch (this.chartType) {
+      case 'bar':
+        return this.i18n.t('js.chart.types.bar');
+      case 'horizontalBar':
+        return this.i18n.t('js.chart.types.horizontal_bar');
+      case 'line':
+        return this.i18n.t('js.chart.types.line');
+      case 'radar':
+        return this.i18n.t('js.chart.types.radar');
+      case 'polarArea':
+        return this.i18n.t('js.chart.types.polar_area');
+      case 'doughnut':
+        return this.i18n.t('js.chart.types.doughnut');
+      case 'pie':
+        return this.i18n.t('js.chart.types.pie');
+      default:
+        return '';
+    }
+  }
```

Comment:
That can be simplified to

```suggestion
  public get chartTypeLabel():string {
    return this.chartType ? this.i18n.t(`js.chart.types.${this.chartType}`) : ''
  }
```